.spectral.yml

```diff
@@ -0,0 +1,2 @@
+extends: spectral:oas
+formats: ["oas3.1"]
```

Comment:
```suggestion
formats: 
- "oas3.1"
```

Reply:
I think this has to be in the brackets - when I remove them I get [an Invalid ruleset provided message](https://github.com/co-cddo/federated-api-model/actions/runs/1545667098), so I put it back